docs/tutorials/index.mdx

```diff
@@ -338,6 +338,25 @@ dates and times.
 </Box>
 </Boxes>
 
+## Services
+An introduction and tutorials on how to communicate between different
+containers with remote-procedure calls (RPCs).
+
+<Boxes>
+<Box title="Services" to="services">
+Introduction to services.
+</Box>
+
+<Box title="Toit" to="services/toit">
+A tutorial on how to communicate between different Toit containers.
+</Box>
+
+<Box title="C" to="services/c">
+A tutorial on how to communicate between Toit and C containers.
```

Comment:
It is pretty crude for now, but it probably makes sense to document what we have.